src/boss/cloudvm/gcp.go

```diff
@@ -69,30 +76,52 @@ func GCPBossTest() {
 	fmt.Printf("Instance: %s\n", instance)
 
 	fmt.Printf("STEP 3: take crash-consistent snapshot of instance\n")
-	disk := instance // assume GCP disk name is same as instance name
-	resp, err := client.Wait(client.GcpSnapshot(disk))
+	disk := instance // assume Gcp disk name is same as instance name
+	start := time.Now()
+	resp, err := client.Wait(client.GcpSnapshot(disk, "test-snap"))
+	snapshot_time := time.Since(start)
 
 	fmt.Println(resp)
 	if err != nil {
 		panic(err)
 	}
 
 	fmt.Printf("STEP 4: create new VM from snapshot\n")
-	resp, err = client.Wait(client.LaunchGCP("test-snap", "test-vm"))
-	fmt.Println(resp)
-	if err != nil {
+	start = time.Now()
+	resp, err = client.Wait(client.LaunchGcp("test-snap", "test-vm"))
+	clone_time := time.Since(start)
+	if err != nil && resp["error"].(map[string]any)["code"] != "409" { //continue if instance already exists error
+		fmt.Printf("instance alreay exists!\n")
+		client.startGcpInstance("test-vm")
+	} else if err != nil {
 		panic(err)
 	}
 
-	fmt.Printf("STEP 5: start worker\n")
-	err = client.StartRemoteWorker()
-	if err != nil {
-		panic(err)
-	}
+	fmt.Printf("\nsnapshot time: %d\n", snapshot_time.Milliseconds())
+	fmt.Printf("clone time: %d\n", clone_time.Milliseconds())
+
+	// fmt.Printf("STEP 5: start worker\n")
```

Comment:
let's delete commented out code

Reply:
Should we consider keeping and uncommenting this section? It tests starting, stopping, and deleting a worker. It might have been commented out while I was conducting experiments and was pushed unintentionally.